examples/inverse/plot_morph_stc_to_fsaverage_dipy.py

```diff
@@ -0,0 +1,231 @@
+
+##############################################################################
+# from examples/inverse/plot_lcmv_beamformer_volume.py
+
+# Setup paths
+data_path = sample.data_path()
+raw_fname = data_path + '/MEG/sample/sample_audvis_raw.fif'
+event_fname = data_path + '/MEG/sample/sample_audvis_raw-eve.fif'
+fname_fwd = data_path + '/MEG/sample/sample_audvis-meg-vol-7-fwd.fif'
+
+# Get epochs
+event_id, tmin, tmax = [1, 2], -0.2, 0.5
+
+# Setup for reading the raw data
+raw = mne.io.read_raw_fif(raw_fname, preload=True)
+raw.info['bads'] = ['MEG 2443', 'EEG 053']  # 2 bads channels
+events = mne.read_events(event_fname)
+
+# Set up pick list: gradiometers and magnetometers, excluding bad channels
+picks = mne.pick_types(raw.info, meg=True, eeg=False, stim=True, eog=True,
+                       exclude='bads')
+
+# Pick the channels of interest
+raw.pick_channels([raw.ch_names[pick] for pick in picks])
+
+# Re-normalize our empty-room projectors, so they are fine after subselection
+raw.info.normalize_proj()
+# Read epochs
+proj = False  # already applied
+epochs = mne.Epochs(raw, events, event_id, tmin, tmax,
+                    baseline=(None, 0), preload=True, proj=proj,
+                    reject=dict(grad=4000e-13, mag=4e-12, eog=150e-6))
+evoked = epochs.average()
+
+# Read regularized noise covariance and compute regularized data covariance
+noise_cov = mne.compute_covariance(epochs, tmin=tmin, tmax=0, method='shrunk')
+data_cov = mne.compute_covariance(epochs, tmin=0.04, tmax=0.15,
+                                  method='shrunk')
+# Read forward model
+forward = mne.read_forward_solution(fname_fwd)
+
+# Compute weights of free orientation (vector) beamformer with weight
+# normalization (neural activity index, NAI). Providing a noise covariance
+# matrix enables whitening of the data and forward solution. Source orientation
+# is optimized by setting pick_ori to 'max-power'.
+# weight_norm can also be set to 'unit-noise-gain'. Source orientation can also
+# be 'normal' (but only when using a surface-based source space) or None,
+# which computes a vector beamfomer. Note, however, that not all combinations
+# of orientation selection and weight normalization are implemented yet.
+filters = make_lcmv(evoked.info, forward, data_cov, reg=0.05,
+                    noise_cov=noise_cov, pick_ori='max-power',
+                    weight_norm='nai')
+
+# Apply this spatial filter to the evoked data.
+stc = apply_lcmv(evoked, filters, max_ori_out='signed')
+
+# take absolute values for plotting
+stc.data[:, :] = np.abs(stc.data)
+
+# Save result in stc files
+stc.save('lcmv-vol')
+
+# stc.crop(0.0, 0.2)
+# Save result in a 4D nifti file
+mne.save_stc_as_volume('lcmv_inverse.nii.gz', stc,
+                       forward['src'], mri_resolution=True)  # set to true
+
+###############################################################################
+# Load nifti data
+
+# load lcmv inverse
+stc = nib.load('lcmv_inverse.nii.gz')
```

Comment:
let's not call this `stc` -- typically this name is reserved for some variant of `SourceEstimate` and here it is a `NIfTI` object (or whatever `nibabel` calls it).

Reply:
hm... any suggestion? nibabel calls their example niftis img... But that's already used by the T1s... what about one of the following:
- src
- img_fct
- img_src
?

Reply:
I would use `img` and call the t1 `t1_img`

Reply:
just seen that I'm stupid...
the T1s are called t1_m or t1_s to refer to moving or static, so I could call the source volume img would this be fine? or would you prefer t1_img_m and t1_img_s?